config/config.sample.php

```diff
@@ -2,7 +2,7 @@
 declare(strict_types=1);
 
 return [
-    'WyriHaximus' => [
+    'Cake' => [
```

Comment:
I think we can drop this as use use `TwigView` as the topmost level.

Reply:
Updated configure code to just use `TwigView`